docs/sphinx/dev_guide/ci.rst

```diff
@@ -12,168 +12,252 @@
 Continuous Integration (CI) Testing
 ************************************
 
-.. important:: * All CI test checks must pass before a pull request can be
-                 merged.
-               * The status (pass/fail and run) for all checks can be viewed by
-                 clicking the appropriate link in the **checks** section of a
-                 GitHub pull request.
+.. important:: * All CI checks must pass before a pull request can be merged.
+               * The status (running and pass/fail) for all checks can be 
+                 viewed by clicking the appropriate link in the **checks** 
+                 section of a GitHub pull request.
 
-The CI tools used by the RAJA project, and which integrate with GitHub are:
+The CI tools used by the RAJA project are:
 
   * **Azure Pipelines** runs builds and tests for Linux, Windows, and MacOS 
     environments using recent versions of various compilers. While we do GPU 
     builds for CUDA, HIP, and SYCL on Azure, RAJA tests are only run for 
-    CPU-only pipelines. See the 
-    `RAJA Azure DevOps <https://dev.azure.com/llnl/RAJA>`_ project to learn 
+    CPU-only pipelines. Docker container images we use for the Linux testing
+    are maintained in the 
+    `RSE Ops Project <https://github.com/rse-ops/docker-images>`_. Please see 
+    the `RAJA Azure DevOps <https://dev.azure.com/llnl/RAJA>`_ project to learn 
     more about our testing there.
 
-  * **Gitlab** instances in the Livermore Computing (LC) Center
-    runs builds and tests in LC resource and compiler environments
-    important to many RAJA user applications. Execution of RAJA CI 
-    pipelines on LC Gitlab resources has restrictions described below. If 
-    you have access to LC resources, you can access additional information about
+  * **GitLab** instances in the Collaboration Zone (CZ) of the Livermore 
+    Computing (LC) Center run builds and tests in LC platform and compiler 
+    environments important to many RAJA user applications. Execution of RAJA CI 
+    pipelines on LC GitLab resources has restrictions, which are described 
+    below. If you have access to LC platforms, you can access additional 
+    information about
     `LC GitLab CI <https://lc.llnl.gov/confluence/display/GITLAB/GitLab+CI>`_
 
-The tools automatically run RAJA builds and tests when a PR is created and 
-when changes are pushed to a PR branch.
+The tools integrate with the RAJA GitHub project and automatically run RAJA 
+builds and tests when a PR is created and when changes are pushed to a PR 
+branch.
```

Comment:
Would it be worth adding a "note" stating that the GitLab CI synchronization, although automated, are asynchronous, which may cause delays?

Reply:
I don't think so. If there are any delays, they pale in comparison to jobs waiting in queues to start running.

If you think it's important to note, it probably should be noted in the RADIUSS shared CI documentation.

Reply:
You are making a point.

Just a remark: While the mirroring is pending the GitLab checks will not appear in the list on GitHub, so you might have a PR seemingly passing all the checks because it is in fact waiting for the GitLab mirroring.

This is not crucial for RAJA because passing the GitLab check is required to merge.